packages/server-wallet/src/protocols/__test__/defund-channel.test.ts

```diff
@@ -31,7 +31,6 @@ beforeEach(async () => {
     '0'
   );
 
-  await DBAdmin.truncateDataBaseFromKnex(knex);
```

Comment:
> So if the database is truncated before all testcases and after each testcase,
we should not need to truncate the database beforeEach testcase.

I have a slight preference truncating in `beforeEach` instead of `afterEach`, since this allows a developer to inspect the DB upon test completion.

Reply:
Good call baf7002ad